seed-job/buildtemplate.yaml

```diff
@@ -112,3 +112,11 @@ parameters:
   displayName: OpenShift namespace
   name: NAMESPACE
   required: true
+- description: From address to be used when sending email
+  displayName: From email address
+  name: FROM_ADDRESS
+  required: true
+- description: SMTP server to use to send emails
+  displayName: SMTP server address
+  name: SMTP_SERVER
```

Comment:
similarly for SMTP server, lets keep default value and also have an option to override.

Reply:
Same as above. I'm not in favor of default values for either of these parameters.